### ISSUE TITLE
About kitti result 

### ISSUE THREAD

**Issue by chenghao-shi** [qinzheng93/GeoTransformer#19]
Hi
    Thanks for sharing the great work! I have problem of reproducing the result on KITTI. I follow the default procedure to evaluate the pre-trained model using default configuration, and here is my result:
RR:0.998, RRE:0.337, RTE:0.063
where RRE is much larger than the paper result. Can you tell me the the possible reason

**Comment by qinzheng93** [comment]
If the data and the weight file is correct, the reasons could be:

1. floating error caused by different GPUs;
2. different random seed (I fix this in the code, so it is less possible);
3. floating error caused by different PyTorch versions, NumPy versions and open3d versions.

I have tested the code on KITTI again, and the results are the same on my machine (2080Ti). Besides, the results on KITTI are already saturated, and could be sensitive to numeric instability.

**Comment by mooncake199809** [comment]
It is caused by the wrong version of the torch. I solve this issue by modifying the version of torch to torch==1.7.1+cu110
